wafer/sponsors/templatetags/sponsors.py

```diff
@@ -11,3 +11,17 @@ def sponsors():
         'sponsors': Sponsor.objects.all().order_by('packages'),
         'packages': SponsorshipPackage.objects.all().prefetch_related('files'),
     }
+
+
+# We use assignment_tag for compatibility with Django 1.8
+# Once we drop 1.8 support, we should change this to
+# simple_tag
+@register.assignment_tag(takes_context=True)
+def sponsor_image_url(context, name):
+    """Returns the corresponding url from the sponsors images"""
+    sponsor = context['sponsor']
```

Comment:
Should we make the context explicit? E.g. `{% sponsor_image_url sponsor, "logo.png" as logo_url %}`?